benchmark/fs/writefile.js

```diff
@@ -0,0 +1,44 @@
+// Call fs.writeFile over and over again really fast.
+// Then see how many times it got called.
+// Yes, this is a silly benchmark.  Most benchmarks are silly.
```

Comment:
[not very thought out suggestion]
Maybe count bytes?
If I understand this correctly this is supposed to benchmarks `node` mechanics not the OS, so number of runs would be my intuition too, but then there is weak correlation between the `len` arg to the results.
If you count bytes, the number of runs can be deduced by calculating `result / len`, so we don't lose information, and it might be easier to see the ratio of `node time / OS time`